docs/en/observability/grant-cases-access.asciidoc

```diff
@@ -17,20 +22,23 @@ a|
 * `All` for the *Cases* feature under *{observability}*.
 * `All` for the *{connectors-feature}* feature under *Management*.
 
-NOTE: Roles without `All` *{connectors-feature}* feature privileges cannot create, add, delete, or modify case connectors.
+[NOTE]
+====
+Roles without `All` *{connectors-feature}* feature privileges cannot create, add, delete, or modify case connectors.
+
+By default, `All` for the *Cases* feature includes authority to delete cases and comments and edit case settings unless you customize the sub-feature privileges.
+====
 
 | Give assignee access to cases
 a| `All` for the *Cases* feature under *{observability}*.
 
 NOTE: Before a user can be assigned to a case, they must log into {kib} at
 least once, which creates a user profile.
 
-| Give view-only access for cases | `Read` for the *Cases* feature under *{observability}*.
-
-| Give access to view and delete cases
```

Comment:
> Give access to view and delete cases

... I deleted this row but if you prefer to keep it separated this way, we'd need to create another new row and cover the "Give access to view cases and edit case settings" scenario.